docs/spec_draft.md

```diff
@@ -3731,6 +3819,50 @@ where `pred_val = rank(pred) == 0 ? pred : pred[i0, ..., iR-1]`.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.send
+
+### Semantics
+
+Sends data `inputs` from one partition to a [stablehlo.recv](#stablehlorecv)
+instruction, in another partition, that shares the same channel handle pair
+{`channel_id`, `channel_type`}.
+
+If `is_host_transfer`, an optional input with default value as `fasle`, is
+`true`, then the operation transfers data to the host.
+
+### Inputs
+
+| Name               | Type                                                               |
+|--------------------|--------------------------------------------------------------------|
+| `inputs`           | variadic number of tensors of any supported type                   |
+| `token`            | `token`                                                            |
+| `channel_id`       | constant of type `si64`                                            |
+| `channel_type`     | enum of `DEVICE_TO_DEVICE`, `DEVICE_TO_HOST`, and `HOST_TO_DEVICE` |
+| `is_host_transfer` | constant of type `i1`                                              |
+
+### Outputs
+
+| Name      | Type  |
+|-----------|-------|
+| `results` | token |
```

Comment:
Typo: "result".

Reply:
For the time, we are having parity with whatever names we have in ODS.
We have https://github.com/openxla/stablehlo/issues/351#issuecomment-1326914173 item 4 to resolve this in bulk as there are [other places](https://github.com/openxla/stablehlo/pull/640#issue-1467186198) as well with similar naming problems.

Reply:
never-mind, I fixed this.